lzma-sys/build.rs

```diff
@@ -74,6 +74,27 @@ fn main() {
         }
     }
 
+    build
```

Comment:
This should be gates behind `OPT_LEVEL`. Only after `OPT_LEVEL = Some("3")`?

Reply:
Related to this MR by name.... `xz2` should also define `NDEBUG` when `OPT_LEVEL` is `3`

https://github.com/xz-mirror/xz/blob/b69da6d4bb6bb11fc0cf066920791990d2b22a06/dos/config.h#L136

Reply:
> Related to this MR by name.... xz2 should also define NDEBUG when OPT_LEVEL is 3

I've added `#define NDEBUG 1` to `config.h`.

Reply:
> This should be gates behind `OPT_LEVEL`. Only after `OPT_LEVEL = Some("3")`?

I'm not sure which line are you referring to.
The line you referenced does not contain anything related to opt-level or optimization flags.